docs/assets/funfacts.txt

```diff
@@ -15,18 +15,18 @@ Grass is a plant with narrow leaves growing from the base. [...]\nGrasses are mo
 SkyClient was once exclusive to Windows users.\nFortunately, now we have a Java-based version, and a web-based version [WIP].
 You'll never believe this, but our Discord's support channel\nis called #support! Use it as necessary.
 SBE has skidded from countless mods including DSM, Skytils and NEU!
-SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient.
+SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient, so the child authors wouldn't get mad.
 SkyClient is for more than Skyblock, even though it started out that way.
-SkyClient is a zero profit organization; we have 40 unpaid volunteers around the globe.
-Most mod's settings are located in a command you can run when in game;\ne.g Patcher is configurable with `/patcher`.
+SkyClient is a non-profit organization; we have 40 unpaid volunteers around the globe.
+Most mods settings are located in a command you can run when in game;\ne.g Patcher is configurable with `/patcher`.
```

Comment:
"mods'"?

Reply:
Most “mod is” settings

Vs

Most mods settings


Grammar